utils/image_saver.py

```diff
@@ -0,0 +1,144 @@
+import os
+import ctypes
+import platform
+from pathlib import Path
+from typing import List, Union, Dict, Optional, AnyStr, Any
+
+from utils import downloader
+from PyInquirer import prompt
+from colorama import Fore, Style
+from utils.image_viewer import tid_repo_prompt
+
+
+def save_image() -> None:
+    """
+    sets the image as the desktop wallpaper
+    If a link is given then first download the image
+    """
+
+    questions: List[Dict[str, Union[List[str], str]]] = [
+        {
+            "type": "list",
+            "name": "download_type",
+            "message": "Set wallpaper from: ",
+            "choices": [
+                "TID Repo",
+                "Link",
+                "Path"
+            ]
+        }
+    ]
+
+    answer: Dict = prompt(questions)["download_type"]
+
+    if answer != 'Path':
+        link: Optional[str] = tid_repo_prompt(
+            "https://raw.githubusercontent.com/Muhimen123/TID/main/image_data.json"
+        ) if answer == 'TID Repo' else input("Please enter download link: ")
+
+        path: str = "//" + os.path.join(os.getcwd(), "data", "images")
+        downloader.download_image(link, path, "current_desktop_wallpaper.png")
+        wallpaper_path: str = f"//{path}current_desktop_wallpaper.png"
```

Comment:
Looks like you misplaced `//` here. Should it go in between `{path}` and `curren...`